examples/src/main/python/streaming/kafka_wordcount.py

```diff
@@ -23,7 +23,7 @@
  http://kafka.apache.org/documentation.html#quickstart
 
  and then run the example
-    `$ bin/spark-submit --driver-class-path external/kafka-assembly/target/scala-*/\
+    `$ bin/spark-submit --jars external/kafka-assembly/target/scala-*/\
```

Comment:
Yes, python Kafka API is added in 1.3 and using `--driver-class-path` is not correct, need to change to `--jars`.